message/message.go

```diff
@@ -218,3 +218,21 @@ func (nc *NetworkCurve) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+const PreAuthEndpoint = "/v1/enduser-auth/preauth"
+
+type PreAuthResponse struct {
+	PollToken string `json:"pollToken"`
+}
+
+const EnduserAuthPoll = "/v1/enduser-auth/poll"
+
+const EnduserAuthPollStatusNotStarted = "NOT_STARTED"
```

Comment:
Do we not need a failure state here? Or would the connection simply close in that case?